src/vm/ast/parser/mod.rs

```diff
@@ -424,6 +499,49 @@ r#"z (let ((x 1) (y 2))
         });
     }
 
+    #[test]
+    fn test_parse_generics() {
+        let input = "<a>";
```

Comment:
Is whitespace allowed?  i.e. `"< a >"`?

Reply:
No, should we?

Reply:
Not sure.  Would developers be surprised of `"< a >"` doesn't work?

Reply:
Preferably avoid the whitespace. Having what looks like a _less-than operator_ in the middle of declared arguments is likely to be confusing rather than helpful. It also complicates the tokenizer and parser, as the semantics of the '<' and '>' will depend on context.